R/property.R

```diff
@@ -403,15 +403,22 @@ as_properties <- function(x) {
 }
 
 as_property <- function(x, name, i) {
-  if (name == "") {
-    msg <- sprintf("`property[[%i]]` is missing a name", i)
-    stop(msg, call. = FALSE)
-  }
 
   if (is_property(x)) {
-    x$name <- name
+    if (is.null(x$name)) {
```

Comment:
So an existing property `$name` has priority over the name supplied in the `list()` call if you supply both, right? I can't tell if that seems right or not, but don't have strong feelings either way

Reply:
Hmmm, I think the list name should win.

Reply:
That was also my gut feeling